tensorflow/models/image/mnist/convolutional.py

```diff
@@ -296,13 +297,15 @@ def eval_in_batches(data, sess):
       # node in the graph it should be fed to.
       feed_dict = {train_data_node: batch_data,
                    train_labels_node: batch_labels}
-      # Run the graph and fetch some of the nodes.
-      _, l, lr, predictions = sess.run(
-          [optimizer, loss, learning_rate, train_prediction],
-          feed_dict=feed_dict)
+      # Run the optimizer.
+      sess.run(optimizer, feed_dict = feed_dict)
+      # print some extra information once reach the evaluation frequency
       if step % EVAL_FREQUENCY == 0:
-        elapsed_time = time.time() - start_time
-        start_time = time.time()
+        # fetch some extra nodes data
```

Comment:
This is not just fetching extra nodes. It's rerunning the graph with these nodes, which means it consumes a batch of data that's not used in training.